### ISSUE TITLE
Cannot delete a table by holding backspace - table remains in viewer and is impossible to delete

### ISSUE THREAD

**Issue by campbellgoe** [nhn/tui.editor#874]
## Describe the bug
When deleting a table by holding the backspace key and deleting it character by character, in markdown mode, the table will still remain visible in the editor and cause further bugs in the viewer.

The console error that was given was this:
Uncaught TypeError: Cannot read property 'columnIdx' of null
    at MarkdownRenderer.t.tableRow (toastui-editor.js:3359)

## To Reproduce
1. Open the default Editor/Viewer in markdown mode
2. Insert a table
3. Click in the markdown editor anywhere after the table markdown
4. Hold backspace until the table is deleted in the editor
5. Note that the table is still visible in the viewer, and that it is not possible to delete.

## Expected behavior
The table should be deleted, even when removing it by holding down backspace in the markdown editor rather than selecting the entire table and then hitting backspace/delete.

## Screenshots
[viewer issue screenshot](https://s3.eu-west-2.amazonaws.com/auth0-5e764dc70efe460c0d46a56b-my-example-project-2000/tui-editor-viewer-issue.png)

## Desktop:
 - OS: Windows 10
 - Browser: Chrome
 - Version: 80

## Additional context
```
toastui-editor.js:3359 Uncaught TypeError: Cannot read property 'columnIdx' of null
    at MarkdownRenderer.t.tableRow (toastui-editor.js:3359)
    at MarkdownRenderer.e.render (toastui-editor.js:3359)
    at toastui-editor.js:5118
    at Array.map (<anonymous>)
    at MarkdownPreview.update (toastui-editor.js:5117)
    at toastui-editor.js:5259
    at forEachOwnProperties (toastui-editor.js:2196)
    at EventManager.emit (toastui-editor.js:5258)
    at MarkdownEditor._emitMarkdownEditorContentChangedEvent (toastui-editor.js:10402)
    at MarkdownEditor._refreshCodeMirrorMarks (toastui-editor.js:10427)
    at toastui-editor.js:10304
    at codemirror.js:2076
    at fireCallbacksForOps (codemirror.js:2033)
    at finishOperation (codemirror.js:2047)
    at endOperation (codemirror.js:3787)
    at HTMLTextAreaElement.<anonymous> (codemirror.js:3924)
```

**Comment by seonim-ryu** [comment]
@campbellgoe I confirmed this issue. Thanks for reporting!

**Comment by seonim-ryu** [comment]
@campbellgoe This issue has been fixed in 2.1.0 and I will close the issue.
